articles/migrations/guides/migration-oauthro-oauthtoken-pwdless.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Migration Guide for Resource Owner Passwordless Authentication
+description: Learn how to migrate your Passwordless API calls and responses from /oauth/ro to /oauth/token
+toc: true
+contentType:
+  - concept
+useCase:
+  - secure-an-api
+  - migrate
+---
+# Migration Guide for Resource Owner Passwordless Credentials Exchange
+
+Support for Resource Owner Password was added to [/oauth/token](/api/authentication#authorization-code). Usage of the [/oauth/ro](/api/authentication#resource-owner) endpoint was deprecated on July 08, 2017. This endpoint was used to exchange an OTP received by the end-user by email or SMS with for an `id_token` and an `access_token`. 
+
+We have implemented a new API that replaces `oauth/ro` for this use case, and we recommend customers to migrate to the new implementation.
+
+## Does this affect me ?
+
+This guide is for users who use the resource owner passwordless credentials exchange, and call `/oauth/ro` directly, without the use of any Auth0 libraries or SDKs. 
+
+## Changes to requests
+
+Previously, the payload of a request to `/oauth/ro` looked similar to this:
+
+```json
+{
+  "grant_type": "password",
+  "client_id": "123",
+  "username": "alice",
+  "password": "A3ddj3w", 
+  "connection": "my-database-connection",
+  "scope": "openid email favorite_color offline_access",
+  "device": "my-device-name"
+}
+```
+
+These are the changes for the new implementation:
+
+* The endpoint to execute token exchanges is now `/oauth/token`
+* [Auth0's own grant type](/api-auth/tutorials/password-grant#realm-support) is used to authenticate users from a specific connection (or `realm`). 
```

Comment:
Not sure if the link to the password grant helps here, because this request uses a different password grant. I think it would also help if we clarified that the **Passwordless OTP** grant type needs to be enabled for the app (**Application | Settings | Show Advanced Settings | Grant Types**), and that the application type needs to be Regular Web or Native.